static/docs/tutorials/basics.md

```diff
@@ -0,0 +1,22 @@
+# DVC Basic Concepts and Features
```

Comment:
can we name all interactive with some prefix and/or suffix `Interactive: Basics` or `Basics (Interactive)` - especially in the sidebar itself